src/DynamoCore/Logging/AnalyticsService.cs

```diff
@@ -17,8 +17,33 @@ class AnalyticsService
         /// the Analytics service and application life cycle start is tracked.
         /// </summary>
         /// <param name="model">DynamoModel</param>
-        internal static void Start(DynamoModel model)
+        /// <param name="disableADPForProcessLifetime">Pass true to disable ADP for the lifetime of the Dynamo or host process</param>
```

Comment:
`disableADP` + descriptive might be easier on the eyes :)

Reply:
@pinzart90 do you mean change the name to `disableADP` and provide a good description?

Reply:
yes...just a thought